src/client/components/pages/entity-revisions.js

```diff
@@ -24,14 +25,28 @@ const Col = bootstrap.Col;
 const ListGroup = bootstrap.ListGroup;
 const ListGroupItem = bootstrap.ListGroupItem;
 
+/**
+* The class is derived from the React Component base class and
+* renders the 'Entity Revisions' page.
+*/
 class EntityRevisions extends React.Component {
 
+/**
+* Binds the class methods to their respective data.
+* @constructor
+* @param {object} props - Properties passed to the component
```

Comment:
This `@param` is formatted correctly but the `renderRevsion` so I'm changing that now. Properties hasn't been purposely uppercased. Should I lowercase it?